src/phase-2/OracleLib.sol

```diff
@@ -0,0 +1,84 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+pragma solidity ^0.8.10;
+
+import "forge-std/console.sol";
+
+import {AggregatorV3Interface} from "../interfaces/chainlink/AggregatorV3Interface.sol";
+import {IwstETH} from "../interfaces/lido/IwstETH.sol";
+import {FixedPointMathLib} from "solmate/utils/FixedPointMathLib.sol";
+import {AccessControl} from "openzeppelin-contracts/access/AccessControl.sol";
+import {scWETHv2} from "./scWETHv2.sol";
+import {IAdapter} from "../scWeth-adapters/IAdapter.sol";
+
+import {CallerNotAdmin, ZeroAddress} from "../errors/scErrors.sol";
+
+contract OracleLib is AccessControl {
```

Comment:
I have a feeling that somehow this is not needed as a separate contract, it could be merged with the helper contract. wdyt?